rtic/Cargo.toml

```diff
@@ -31,24 +31,22 @@ features = ["rtic-macros/test-template"]
 name = "rtic"
 
 [dependencies]
+# riscv-slic = { path = "../../riscv-slic/riscv-slic", optional = true }
+riscv-slic = { git = "https://github.com/romancardenas/riscv-slic.git", rev = "2a91edb", optional = true }
 esp32c3 = { version = "0.20.0", optional = true}
 riscv = {version = "0.11.0", optional = true}
 cortex-m = { version = "0.7.0", optional = true }
 bare-metal = "1.0.0"
-#portable-atomic = { version = "0.3.19" }
+# portable-atomic = { version = "0.3.19" }
```

Comment:
Not used at all, do we need it? :thinking: @korken89

Reply:
I think it is a "reminder" that we should move from atomic-polyfill to portable-atomic :D